packages/relottie-metadata/__tests__/index.test.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2024 Design Barn Inc.
+ */
+
+import { readdirSync, readFileSync } from 'fs';
+import path from 'path';
+
+import relottieParse from '@lottiefiles/relottie-parse';
+import relottieStringify from '@lottiefiles/relottie-stringify';
+import { toMatchSpecificSnapshot } from 'jest-specific-snapshot';
+import { unified } from 'unified';
+
+import relottieMetadata from '../src/metadata.js';
+
+expect.extend({
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  toMatchDecoratedSpecificSnapshot(received, snapshotFile) {
+    // You can modify received data or create dynamic snapshot path
+    return toMatchSpecificSnapshot.call(this, received, snapshotFile, '');
+  },
+});
+
+const FIXTURE_DIR_PATH = '../../__fixtures__/';
+const SNAPSHOT_DIR_PATH = './__snapshots__/';
+
+describe('relottie-metadata', () => {
+  describe('fixtures', () => {
+    const FEATURES_DIR_NAME = 'features';
+    const EXTRA_DIR_NAME = 'extra';
+
+    describe(FEATURES_DIR_NAME, () => {
+      const dirPath = `${FIXTURE_DIR_PATH + FEATURES_DIR_NAME}/`;
+      const snapshotDirPath = `${SNAPSHOT_DIR_PATH + FEATURES_DIR_NAME}/`;
+
+      const files = readdirSync(dirPath);
+
+      files.forEach((file) => {
```

Comment:
[Suggestion]
you may consider using `test.each`
https://jestjs.io/docs/api#testeachtablename-fn-timeout